main.js

```diff
@@ -0,0 +1,95 @@
+let myLibrary = [
```

Comment:
prefer `const` over `let` if you are not reassigning it.

Reply:
Resolved